.github/workflows/build-android.yml

```diff
@@ -21,6 +21,12 @@ jobs:
         run: |
           mkdir -p ./assets/messaging
           echo $SERVER_TOKEN > ./assets/messaging/cloud_messaging.json
+      - name: fetch mailer credentials
+        env:
+          CREDENTIALS: ${{ secrets.MAILER_CREDENTIALS }}
```

Comment:
please give it a more specific name as `EMAIL_CREDENTIALS: ${{ secrets.EMAIL_CREDENTIALS }}`